include/tvm/relax/expr.h

```diff
@@ -338,7 +338,7 @@ class TupleGetItemNode : public ExprNode {
   /*! \brief The tuple Expression */
   Expr tuple;
   /*! \brief which value to get */
-  int index;
+  Expr index;
```

Comment:
For the symbolic fetching case, we should use a lower level instrinsic. This is mainly to simplify the overall constant accessing of the tuple elements(which is the common case)